### PR TITLE
remove locale_disambiguation column

### LINKED ISSUE #9605: Remove `locale_disambiguation` column from the `addons` table
Once #9522 is merged, we can remove the `locale_disambiguation` column from the `addons` table (it will have been removed from the model).

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #9605

### CODE REVIEW

File: src/olympia/migrations/1049-remove-locale-disambiguation-column.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `addons`
+ DROP COLUMN `locale_disambiguation`;
+
```
Comment (eviljeff) [inline src/olympia/migrations/1049-remove-locale-disambiguation-column.sql]: new line at the end of the file please

Comment (championshuttler) [reply src/olympia/migrations/1049-remove-locale-disambiguation-column.sql]: @eviljeff fixed :)

**Comment by EnTeQuAk** [comment]
@diox I guess the 'do not merge' label was to not merge it into this tag but the next one? Just making sure before we merge this :)

**Comment by diox** [comment]
Yep!

**Comment by EnTeQuAk** [comment]
let's get the money then :)

**Pull request merged**